### DISCUSSION TITLE
stage.obs(config)

### DISCUSSION THREAD

**Discussion by guillaumevernieres** [NOAA-EMC/GDASApp#183] [General]
@RussTreadon-NOAA , @CoryMartin-NOAA , unless I miss-grepped ... I think I'm the only one using that function:
https://github.com/NOAA-EMC/GDASApp/blob/2712a22aa81788d85e14cac358de12956a75d238/ush/ufsda/stage.py#L315
Am I missing something?

I need to modify it or, if used somewhere else, write a new one that will allow me to do a slightly smarter fetch to resolve #182

**Comment by RussTreadon-NOAA** [comment]
Agreed, @guillaumevernieres .  I think it is safe for you to modify the `obs` function.
